src/common/iop_order.c

```diff
@@ -1412,34 +1539,13 @@ static void _ioppr_check_rules(GList *iop_list, const int imgid, const char *msg
   if(fences) g_list_free(fences);
 }
 
-// how is on-the-fly conversion done
-// Currently a hack to support v3 history to later
-// returns the history version of imgid
-int dt_ioppr_convert_onthefly(const int imgid)
+// migrate to another iop_order version the given image
+// not that this is actually a non exported routine but it will
```

Comment:
1. migrate the given image to another iop_order version
2. note that this is :-) First i misunderstood

Reply:
Fixed.